Pin Rust Nightly to 2020-10-25

repi opened the pull request:
Fix #137

bjorn3 commented on setup.bat:
```diff
@@ -1,3 +1,3 @@
 setlocal
 
-rustup toolchain install nightly --component rust-src rustc-dev llvm-tools-preview
+rustup toolchain install nightly-2020-10-24 --component rust-src rustc-dev llvm-tools-preview
```
Wouldn't `rustup component add rust-src rustc-dev llvm-tools-preview` automatically install the correct nightly?

repi replied:
Probably, if it does read the `rust-toolchain` file, worth trying

khyperia commented:
Oh, uh, didn't see this in my pile of inbox before I already pushed a fix to get it to compile on latest nightly.

repi commented:
I can update this pinning to the date of the latest nightly

khyperia commented:
sounds good! I don't want to r+ this because then it'd automerge, and fail because the CI run now is out of date, but, pseudo-r+ from me for pinning to latest nightly :stuck_out_tongue:

repi commented:
Thx, will update this

repi commented:
@khyperia rebased this to latest, updated to use 2020-10-25 instead and changed current setup scripts to just add the components in general to bjorn3's suggestion instead of to a specific nightly version. Makes it a bit easier.

So if you approve this one right now it should be merged automatically if tests pass

bjorn3 commented on setup.bat:
```diff
@@ -1,3 +1,3 @@
 setlocal
 
-rustup toolchain install nightly-2020-10-24 --component rust-src rustc-dev llvm-tools-preview
+rustup component add rust-src rustc-dev llvm-tools-preview
```
nit: missing trailing newline

bjorn3 commented on setup.sh:
```diff
@@ -1,2 +1,2 @@
 #!/usr/bin/env bash
-rustup toolchain install nightly --component rust-src rustc-dev llvm-tools-preview
+rustup component add rust-src rustc-dev llvm-tools-preview
```
nit: missing trailing newline
